src/tools/scripts/nettoyageUserCreationError.js

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env node
+'use strict';
+
+const { program } = require('commander');
+const { execute } = require('../utils');
+
+const getElements = async (db, collection) => await db.collection(collection).find({
+  userCreated: true, userCreationError: true
+});
+
+const getUserByEntity = db => async (id, role) => await db.collection('users').findOne({
+  'entity.$id': id,
+  'roles': { '$in': [role] }
+});
+
+const updateUserCreated = (db, collection) => async id => await db.collection(collection).updateOne(
+  { '_id': id },
+  { '$set': { 'userCreated': false } }
+);
+
+const deleteUserCreationError = (db, collection) => async id => await db.collection(collection).updateOne(
+  { '_id': id },
+  { '$unset': { 'userCreationError': '' } }
+);
+
+program.option('-f, --fix', 'fixer en base les statuts userCreated et userCreationError');
+program.option('-c, --collection <collection>', 'choisir d\'appliquer les corrections sur les conseillers ou sur les structures');
+program.parse(process.argv);
+
+execute(__filename, async ({ exit, logger, db }) => {
+  logger.info('Début du script de nettoyage des incohérences sur userCreationError');
+  const { fix, collection } = program;
+
+  const elements = await getElements(db, collection);
+
+  if (collection === 'structures') {
```

Comment:
ok tu peux faire un early return en checkant collection  in structures ou conseillers

Reply:
pour sortir en erreur dès le départ